build.sbt

```diff
@@ -220,6 +218,7 @@ lazy val cli = project
   .in(file("scip-java"))
   .settings(
     moduleName := "scip-java",
+    resolvers ++= Resolver.sonatypeOssRepos("snapshots"),
```

Comment:
Will be removed once I release Moped (after making sure everything in this PR works)